html-practice/facebook-mock-up.html

```diff
@@ -0,0 +1,179 @@
+<!DOCTYPE HTML>
+<html>
+	<head>
+		<style>
+		html {
+			background-color: #e9eaed;
+			height: 100%;
+			width: 100%;
+		}
+		body {
+			width: 100%;
+			height: 800px;
+			margin: 0;
+		}
+		#Facebook-menu-bar {
```

Comment:
For consistency's sake, I'd follow the `lower-case-with-hyphens` naming convention here:

i.e. `#facebook-menu-bar`